components/base/agilex/limo_base.go

```diff
@@ -29,8 +29,9 @@ var (
 // default port for limo serial comm.
 const (
 	defaultSerialPath  = "/dev/ttyTHS1"
-	minTurningRadiusM  = 0.4 // from datasheet at: https://www.wevolver.com/specs/agilex-limo
-	defaultBaseTreadMm = 172 // "Tread" from datasheet at: https://www.wevolver.com/specs/agilex-limo
+	minTurningRadiusM  = 0.4           // from datasheet at: https://www.wevolver.com/specs/agilex-limo
+	defaultBaseTreadMm = 172           // "Tread" from datasheet at: https://www.wevolver.com/specs/agilex-limo
+	defaultBaseWidthM  = 220.0 * 0.001 // Width from engineering drawing athttps://docs.trossenrobotics.com/agilex_limo_docs/specifications/limo.html
```

Comment:
@raybjork, one thing while I was self-reviewing, the scope doc doesn't mention which "width" nav service is going to consume. The wheelbase is what was mentioned in the TP space paper, and `width_mm` is the wheelbase used in the wheeled driver.

The overall width of the base is larger, if it's expected that this parameter be used for obstacles, I can commit these dimensions. I have a feeling for wheeled - no since the `base.CollisionGeometry` takes care of that for built-in motion obstacle avoidance. Let me know if it is in fact the overall width and I'll add a checker in `wheeled` as well for the different frame configs (started there but didn't go too far) 

Made this a separate commit so I can revert it easily.

Reply:
We don't have any plans to use width_mm from GetProperties in motion planning as part of this scope. I think Eliot just thought it should also be exposed via GetProperties. For collisions, motion planning will use the overall width of the base, which it gets from the frame system, I believe.

Reply:
width_mm should still be exposed via GetProperties. I'd defer to @raybjork on whether the modules need to have frame configs, since I'm not sure.

Reply:
yeah width_mm is still exposed, I was reverting the cases where I  expose a different dimension than the wheelbase.